### PR TITLE
[tags & users & libs & beez] Always use htmlspecialchars($str, ENT_COMPAT, 'UTF-8')

### PR DESCRIPTION
**Pull request by zero-24**
Pull Request for Issue #10399 .
#### Summary of Changes

Always use htmlspecialchars($str, ENT_COMPAT, 'UTF-8')
#### Testing Instructions

Please do a code review or test all changed files.

### CODE REVIEW

File: templates/beez3/html/com_newsfeeds/category/default_items.php [hunk 1]
```diff
@@ -90,14 +90,14 @@
...
-	<?php if ($this->params->def('show_pagination_results', 1)) : ?>
-		<p class="counter">
-			<?php echo $this->pagination->getPagesCounter(); ?>
-		</p>
-	<?php endif; ?>
-	<?php echo $this->pagination->getPagesLinks(); ?>
-	</div>
+		<div class="pagination">
+		<?php if ($this->params->def('show_pagination_results', 1)) : ?>
+			<p class="counter">
```
Comment (brianteeman) [inline templates/beez3/html/com_newsfeeds/category/default_items.php]: is this extra tab correct?

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on f3466180b7fdd53af15ee082d78498284e9328d8<br /><br />on review <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10440">issues.joomla.org/joomla-cms/10440</a>.</sub>

File: templates/beez3/html/com_newsfeeds/category/default_items.php (hunk 1 above)
Comment (zero-24) [inline templates/beez3/html/com_newsfeeds/category/default_items.php]: Yes because itbis in a if see line 92 ;)

Comment (brianteeman) [inline templates/beez3/html/com_newsfeeds/category/default_items.php]: so shouldnt that be
if
tab div
tab tab if
tab tab tab p

Comment (zero-24) [inline templates/beez3/html/com_newsfeeds/category/default_items.php]: no as the div is on the same hirachie than the if ;)

so
if
tab div
tab if
tab tab p

is correct ;)

Comment (brianteeman) [inline templates/beez3/html/com_newsfeeds/category/default_items.php]: Seems wrong to me - but what do I know ;)

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on f3466180b7fdd53af15ee082d78498284e9328d8<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10440">issues.joomla.org/joomla-cms/10440</a>.</sub>

**Comment by brianteeman** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10440">issues.joomla.org/joomla-cms/10440</a>.</sub>

**Comment by andrepereiradasilva** [comment]
```
FILE: ...build/joomla/joomla-cms/libraries/joomla/database/exporter/pdomysql.php
--------------------------------------------------------------------------------
FOUND 0 ERROR(S) AND 1 WARNING(S) AFFECTING 1 LINE(S)
--------------------------------------------------------------------------------
 79 | WARNING | Line exceeds 150 characters; contains 154 characters
--------------------------------------------------------------------------------
```

**Comment by andrepereiradasilva** [comment]
ok ignore my last comment the travis errors as nothing to do with your PR.

**Comment by brianteeman** [comment]
We always ignore all your comments ;)

Cant wait to share a beer or three next week

On 13 May 2016 at 21:55, andrepereiradasilva <notifications@github.com>
wrote:

> ok ignore my last comment the travis errors as nothing to do with your PR.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/joomla/joomla-cms/pull/10440#issuecomment-219156796>
>



-- 
Brian Teeman
Co-founder Joomla! and OpenSourceMatters Inc.
http://brian.teeman.net/

**Comment by zero-24** [comment]
I have fixed the codestyle issue. I hope travis is happy than ;)

**Pull request merged**